eslint-plugin-expensify/tests/no-api-side-effect-actions.test.js

```diff
@@ -0,0 +1,89 @@
+const RuleTester = require('eslint').RuleTester;
+const rule = require('../no-api-side-effect-actions');
+const message = require('../CONST').MESSAGE.NO_API_SIDE_EFFECTS_ACTIONS;
+
+const ruleTester = new RuleTester({
+    parserOptions: {
+        ecmaVersion: 6,
+        sourceType: 'module',
+    },
+});
+
+ruleTester.run('no-api-side-effect-actions', rule, {
```

Comment:
It would be more direct to call the rule something like `no-call-actions-from-actions` or `prefer-one-one-one-design`.

Reply:
`no-call-actions-from-actions` sounds good to me!